generate_certificate.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 # Fully qualified name of the CSR object
-csr="certificatesigningrequests.v1beta1.certificates.k8s.io"
```

Comment:
If You will be also so kind and update the change in https://github.com/criblio/k8s-webhook-cert-manager/blob/0f4255fbcae051f953e0da22c626275794c7e432/e2e-tests/manifests/webhook.yaml#L1 then we can get rid of `beta1` part completely

Reply:
done.